### PR TITLE
unblocks AP interface on ubuntu

### PR DESCRIPTION
**Pull request by lvrach**
https://askubuntu.com/questions/472794/hostapd-error-nl80211-could-not-configure-driver-mode

Solving the following issue:

```
[*] Starting the fake access point...
[+] Driver initialization failed! (hostapd error)
[+] Try a different wireless interface using -aI option.

[!] Closing
```

### CODE REVIEW

File: wifiphisher/pywifiphisher.py [hunk 1]
```diff
@@ -746,6 +746,37 @@ def get_hostapd():
...
+            ['rfkill', 'unblock', 'wlan'],
+            stdout=PIPE,
+            stderr=DN
+        ).wait()
+
+        time.sleep(1)
+
+        for interface in interfaces:
+            Popen(
+                ['ifconfig', interface, '10.15.0.1/24', 'up'],
```
Comment (sophron) [inline wifiphisher/pywifiphisher.py]: Setting the IP address here troubles me. I'll figure out why exactly this is needed.

Comment (lvrach) [inline wifiphisher/pywifiphisher.py]: I've done some tests and they prove that is not necessary .

Comment (wraith-wireless) [inline wifiphisher/pywifiphisher.py]: Shouldn't have to use nmcli and rfkill. Deleting all interfaces on the phy before creating a new device on that phy should remove all problems with NetworkManager interfering with the new device

Comment (sophron) [inline wifiphisher/pywifiphisher.py]: @lvrach Good.

@wraith-wireless Thanks for the notice. As soon as we have PyRIC imported, we'll switch to that.

**Pull request merged**